src/Components/CreateImageWizard/formComponents/FileSystemConfigButtons.js

```diff
@@ -16,16 +16,16 @@ const FileSystemConfigButtons = ({ handleNext, handlePrev, nextStep }) => {
   );
   const [nextHasBeenClicked, setNextHasBeenClicked] = useState(false);
   const prefetchArchitectures = imageBuilderApi.usePrefetch('getArchitectures');
+  const errors = getState()?.errors?.['file-system-configuration'];
 
   useEffect(() => {
-    const errors = getState()?.errors?.['file-system-configuration'];
     errors ? setHasErrors(true) : setHasErrors(false);
 
     if (!errors) {
       setNextHasBeenClicked(false);
       change('file-system-config-show-errors', false);
     }
-  });
+  }, [errors, change]);
```

Comment:
Use this pattern:
https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

Reply:
as we discussed in the group channel, we agreed to add those dependencies
and in the future maybe we will refector the code and remove the useEffect hook were ever we can